ROLES.md

```diff
@@ -0,0 +1,99 @@
+	* Must enable 2FA on their Github account
+* Privileges
+  * Entitled to vote in the Steering Committee Election
+  * Entitled to stand in the Steering Committee Election
+  * Can recommend other contributors to become Org Members
+  * May be assigned Issues and Reviews
+  * May give commands to CI/CD automation
+  * Can be added to repository teams
+  * Added as a member of the Github org
+
+The process for a Contributor to become a Member is as follows:
+
+1. Open an GitHub Issue against the [**TODO:** prometheus/community repo]
+   1. Ensure your sponsors are @mentioned on the issue
+   2. Assign label **TODO**
+   3. Complete every item on the checklist (**TODO:** template)
+   4. Make sure that the list of contributions included is representative of your work on the project.
+2. Sponsors reply via a comment on the GitHub Issue with explicit confirmation
+3. Once Sponsors have responded positively, the request will be reviewed by the Steering Committee or its delegates to make sure certain contributions are valid and non-trivial.
+4. If the threshold is met, the Contributor is added to [https://github.com/orgs/prometheus](https://github.com/orgs/prometheus)/people and [https://github.com/orgs/prometheus-community/people](https://github.com/orgs/prometheus-community/people) and becomes a Member
+
+### Maintainer
+
+Description: Maintainers are well-established contributors who review and approve code contributions. Maintainers are responsible for the long-term health of the project including reviewing PRs, stability, adoption, security, etc.
+
+Maintainer status is scoped to a repository or a part of the codebase.
+
+A Maintainer must meet the responsibilities and requirements of an Organization Member, plus:
+
+* Responsibilities include
+  * Demonstrate sound technical judgment (maybe asked to step down by a maintainer if they lose the confidence of the other maintainers)
+  * Responsible for project quality control via code reviews
+  * Responsible for technical quality control of the documentation, related to their project
+  * Expected to be responsive to review requests
+  * Mentoring new contributors
+  * Determining strategy and policy for the project
+  * Participating in, and leading, community meetings
+  * May merge PRs
+* Requirements
+  * Consistently contributing to the codebase
+  * Reviewer for or author of substantial PRs to the codebase, with the definition of substantial subject to the maintainer's discretion (e.g. refactors/adds new functionality rather than one-line pulls).
+  * Demonstrates a broad knowledge of the project across multiple areas
+  * Can exercise judgment for the good of the project, independent of their employer, friends, or team
+  * Mentors other contributors
+* Additional privileges
+  * Represent the project in public as a Maintainer
+  * Have a vote in Maintainer decision-making meetings
```

Comment:
One thing not defined here is access to sensitive information like CI/CD secrets, passwords, etc that a repo might need for CI/CD, docker image publishing, etc, and who manages those sensitive items. I think it would be good to spell that out.